com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -55,9 +55,9 @@ internal BitWriter BeginSendServerRpc(ServerRpcSendParams sendParams, bool isRel
             }
 
             var writer = rpcQueueMananger.BeginAddQueueItemToOutboundFrame(RPCQueueManager.QueueItemType.ServerRpc, Time.realtimeSinceStartup, StandardRpc_ChannelName, 0, NetworkingManager.Singleton.ServerClientId, null);
-            writer.WriteBit(false); // Encrypted
-            writer.WriteBit(false); // Authenticated
-            writer.WriteBits(MLAPIConstants.MLAPI_SERVER_RPC, 6); // MessageType
+//            writer.WriteBit(false); // Encrypted
```

Comment:
Do you mean to leave this commented code in?

Reply:
yep. Will update.